c2cgeoportal/lib/cacheversion.py

```diff
@@ -61,7 +61,8 @@ def __call__(self, request):
 
         response = self.handler(request)
 
-        response.headers["Access-Control-Allow-Origin"] = "*"
-        response.headers["Access-Control-Allow-Headers"] = "X-Requested-With, Content-Type"
+        if path[1] in CACHE_PATH:
```

Comment:
i.-e.:
the `request.path_info` can have the value: `"/proj-ngeo/5f5e11b65b774343945a44eea2882d5d/build/mobile.css"`
the path will have the value `['', 'proj-ngeo', ...]`
the `CACHE_PATH` will have the value `[u'static', u'project', u'proj', u'proj-ngeo']`
...
For the dynamic view We can ash for user data and credentials, than we needs something dynamic that done here: https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/lib/caching.py#L109-L155

Bonus: to have the response :-)